### PR TITLE
Tox cleanup

### PR DESCRIPTION
**Pull request by wsanchez**
Minor cleanup of `tox.ini`, including a round of requirement updates that `requires.io` doesn't seem to notice.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -496,7 +462,7 @@ recreate = true
 deps =
 
 commands =
-    pip freeze
```
Comment (adiroiban) [inline tox.ini:499]: the `dependencies` seems to me like overengineering .... create a new virtualenv just to run pip freeze ? 

Is the list of dependencies reviewed on a regular basis?

Comment (adiroiban) [inline tox.ini:499]: Is this env needed?

I don't see it called as part of CI workflows

**Review by adiroiban — APPROVED** [review]
Looks good. Thanks. Only minor comments.

File: tox.ini (hunk 1 above)
Comment (wsanchez) [reply tox.ini:499]: Yes, I use this to update the dependency list periodically.

File: tox.ini [hunk 2]
```diff
@@ -396,7 +362,7 @@ description = build documentation
 basepython = {[default]basepython}
```
[the remark this answers is not in the document]
Comment (wsanchez) [reply tox.ini:362]: Only if you want to be sure that each env is consistently built with the same version of Python. YYMV.

**Comment by wsanchez** [comment]
Thanks @adiroiban

**Pull request merged**